backend/app/solr/operations.py

```diff
@@ -0,0 +1,31 @@
+"""Operations on Solr"""
+from httpx import AsyncClient, Response
+
+from app.config import SOLR_URL
+
+
+async def search(
+    client: AsyncClient,
+    collection: str,
+    *,
+    q: str,
+    qf: list[str],
+    fq: list[str],
+    sort: list[str],
+    rows: int,
+    cursor: str = "*",
+) -> Response:
+    """Retrieve search results"""
```

Comment:
This is the redundant comment I was talking about yesterday. Code is enough self-explanatory.